### DISCUSSION TITLE
Backspace suddenly overreacting

### DISCUSSION THREAD

**Discussion by ScyMarshall** [kmonad/kmonad#637] [Q&A]
Hi,
my kbd works fine so far, but suddenly bspc triggers too hard an erases too much.
I thought maybe it was a faulty button, remapped, same result. Outside kmonad all works fine.
Tried forcing a pause like:
@bspc 
#(bspc P5)
did not work.
What next?

**Comment by gerhard-h** [comment]
maybe a ctrl key is 'stuck'?

if not you could try to tweak key repeat globaly
```
(defcfg

  ;; For Windows
     input  (low-level-hook)
     output (send-event-sink 500 30)
     ;; output now has two more arguments, both in milliseconds. The first defining how long before key-repeat starts, the second how long between each repeat event.
```

**Reply by ScyMarshall** [reply]
Hi, it is no stuck,i remapped other keys with same result... And i allready have the 500 30 nerf.

**Reply by gerhard-h** [reply]
does it feel different when you use 
output (send-event-sink 1000 1000) or does it get ignored?

**Reply by ScyMarshall** [reply]
Oh,sorry.Ill try it tomorrow
